cmake/sample_defs/cpu1_platform_cfg.h

```diff
@@ -1786,7 +1776,7 @@
 **  \par Limits
 **       This value can be any 32 bit unsigned integer.
 */
-#define CFE_PLATFORM_TBL_VALID_PRID_1            (CFE_PLATFORM_CPU_ID)
+#define CFE_PLATFORM_TBL_VALID_PRID_1            (10)
```

Comment:
Shouldn't this come from the target configuration?

Reply:
In this case - no, I don't think so....

This particular item is an optional extra table validation step, and it need not match the actual processor ID of the CFE instance running TBL services.  The code is actually compiled out by default and its been arguably broken for a while, due to the discrepancies outlined in #710 

If the TBL services feature is always supposed to allow the current/active processor ID to be loaded, then it must be coded that way by calling CFE_PSP_GetProcessorId().  We shouldn't make users repeat themselves (DRY principle), and more importantly, the value is overridable at runtime via command line options.  

This list of alternative allowable processor IDs should be used only when you specifically want _different_ (non-local) IDs to be capable of being loaded.